docs/snippets/pre_commit.rst

```diff
@@ -0,0 +1,11 @@
+**pydocstyle** can be included as a hook for `pre-commit`_.  The easiest way to get
```

Comment:
Link to pre-commit?

Reply:
It's a named reference. It will be resolved into a link if sphinx know how to resolve it (e.g. via intersphinx from the other sphinx site or any other way), otherwise it'll be rendered as a plain text.